src/qt/intro.h

```diff
@@ -10,6 +10,7 @@
 #include <QThread>
 
 static const bool DEFAULT_CHOOSE_DATADIR = false;
+static constexpr uint64_t GB_BYTES = 1000000000LL;
```

Comment:
I'd propose put this in `guiconstants.h` instead, there is no reason `optionsdialog.cpp` should depend on `intro.h`